### PR TITLE
The latest Cmake 3.28.3 is failing with "Could NOT find Protobuf (missing: Protobuf_LIBRARIES)". Use Cmake 3.27.9

### PR DESCRIPTION
**Pull request by liqunfu**
### Description
https://gitlab.kitware.com/cmake/cmake/-/issues/25704

### CODE REVIEW

**Review by thiagocrepaldi — COMMENTED** [review]
Is `choco` available on the CI?

**Comment by liqunfu** [comment]
> Is `choco` available on the CI?

yes it is - cmake of desired version is installed with the fix.

File: .azure-pipelines/Windows-CI.yml [hunk 1]
```diff
@@ -56,6 +56,13 @@ jobs:
   - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
     displayName: Add conda to PATH
 
+  # cmake 3.28.3 failed to find protobuf installed from conda-forge
```
Comment (gramalingam) [inline .azure-pipelines/Windows-CI.yml:59]: Weird. I wonder why.

**Review by gramalingam — APPROVED** [review]

File: .azure-pipelines/Windows-CI.yml (hunk 1 above)
Comment (gramalingam) [reply .azure-pipelines/Windows-CI.yml:59]: I wonder if something is missing in the cmake files.

Comment (liqunfu) [reply .azure-pipelines/Windows-CI.yml:59]: hope it is resolved in the next cmake version: https://gitlab.kitware.com/cmake/cmake/-/issues/25704

**Review by thiagocrepaldi — APPROVED** [review]

**Pull request merged**